app/models/case/sar/offender.rb

```diff
@@ -408,6 +418,17 @@ def user_dealing_with_vetting
     user_for_vetting
   end
 
+  def next_seq_for_rejected_case_number(received_date)
```

Comment:
This method is not getting called anywhere.

I'm not sure what is being done below is necessary. It should be possible to use the existing method that creates a number in the base class and prefix a letter.